IFC4x3/Properties/t/TrackNumber_04f6CUpMXA8hZooEXNRWkz/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TrackNumber_04f6CUpMXA8hZooEXNRWkz" Name="TrackNumber" UniqueId="04a4631e-cd68-4a22-b8f2-c8e8576e0bbd" PrimaryDataType="IfcIdentifier" />
```

Comment:
Same question as above.

Reply:
One track is a "track system", which has one ID and one Number